CompuCell3D/core/pyinterface/CMakeLists.txt

```diff
@@ -79,19 +51,15 @@ IF(SWIG_FOUND)
         add_definitions(-DVTK6)
     endif()
 
-
     add_subdirectory(PlayerPythonNew)
     add_subdirectory(SerializerDE)
     add_subdirectory(PyPlugin)
     # add_subdirectory(Example)
     # add_subdirectory(Fields)
-if(USE_DOLFIN)
-    add_subdirectory(dolfinCC3D)
-endif(USE_DOLFIN)
+    if(USE_DOLFIN)
+        add_subdirectory(dolfinCC3D)
```

Comment:
we need to bring this back even if it is only on OSX and Linux - just a side remark